src/movement/thruster_sabertooth.cpp

```diff
@@ -22,6 +22,49 @@ void checksum (uint8_t serial_data[4])
   serial_data[3] = (serial_data[0]+serial_data[1]+serial_data[2]) & 127;
 }
 
+void createThrusterPacket (uint8_t serial_data[4], double value, int i)
```

Comment:
A nice thing here would be to have a comment explaining the structure of the packet so that it is here and the data sheet doesn't need to be referenced when changes are needed. Also, `i` isn't a very descriptive variable name.